server/src/project/project.service.ts

```diff
@@ -382,77 +387,85 @@ export class ProjectService {
       }, 1);
     const [firstProject] = projects;
 
-    const transformedProject = await transformProjectAttributes(firstProject);
-
-    // get geoms from carto that match array of bbls
-    const bblsFeaturecollection = await this.geometryService.getBblsFeaturecollection(firstProject.bbls);
-
-    if (bblsFeaturecollection == null) {
-      console.log(`MapPLUTO does not contain matching BBLs for project ${firstProject.id}`);
+    if (projects.length < 1) {
+      throw new HttpException({
+        "code": "PROJECT",
```

Comment:
nit: More specific code and title is better here
```
{
  "code": "PROJECT_NOT_FOUND", 
   "title": "Project not found",
   "detail": `Project ${name} not found`,
   }
```

Reply:
will do!